<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="resultados">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>791</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Resultados por mesas / parejas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ver_mesas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Ver Mesas / Parejas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ver_ranking">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Ver ranking </string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>791</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="crea_campeonato">
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Crea Campeonato</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Inscribe_parejas">
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Inscribe Parejas</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
